js/_website/generate_jsons/src/demos/__init__.py

```diff
@@ -0,0 +1,147 @@
+import os
+import json
+
+GRADIO_DEMO_DIR = os.path.abspath(os.path.join(os.getcwd(), "..", "..", "..", "demo"))
```

Comment:
Same as above, cwd is inconsistent depending on location, lets make this more deterministic.